### ISSUE TITLE
Button checkmark is not centered

### ISSUE THREAD

**Issue by crutchcorn** [papyros/qml-material#338]
![arch linux-2015-11-29-22-34-21](https://cloud.githubusercontent.com/assets/9100169/11465032/29dae8fa-96ea-11e5-870b-494fdc690a5c.png)

**Comment by ricardomv** [comment]
I thought this was fixed recently, is this still occurring in the develop branch?

**Comment by crutchcorn** [comment]
Yes sir. This screenshot you see was build from the develop branch

**Comment by ricardomv** [comment]
What's your qt version, distro, screen resolution, dpi (output of xrandr) and the command you use to start the demo or if you run it from qtcreator. I can't reprduce this on my machine

**Comment by eyelash** [comment]
duplicate of #232

**Comment by IdrissDimson** [comment]
yeah that's right^
